### PR TITLE
support default HostIp

### PR DESCRIPTION
**Pull request by Stevent-fei**
### Describe what this PR does / why we need it
#1787

### CODE REVIEW

File: pkg/infradriver/ssh_infradriver.go [hunk 1]
```diff
@@ -141,11 +142,23 @@ func (d *SSHInfraDriver) GetHostIPListByRole(role string) []net.IP {
...
+
+	return hostEnv
 }
 
 func (d *SSHInfraDriver) GetClusterEnv() map[string]interface{} {
-	return d.clusterRenderData
+	if d.clusterEnv["RegistryDomain"] == nil {
+		d.clusterEnv["RegistryDomain"] = "sea.hub"
+	}
+	if d.clusterEnv["RegistryPort"] == nil {
```
Comment (kakaZhou719) [inline pkg/infradriver/ssh_infradriver.go:157]: use if ok to check its map key, and add more code comments.

File: pkg/infradriver/ssh_infradriver.go [hunk 2]
```diff
@@ -141,11 +142,21 @@ func (d *SSHInfraDriver) GetHostIPListByRole(role string) []net.IP {
...
+	if _, ok := hostEnv["HostIP"]; !ok {
+		hostEnv["HostIP"] = host.String()
+	}
+	return hostEnv
 }
 
 func (d *SSHInfraDriver) GetClusterEnv() map[string]interface{} {
-	return d.clusterRenderData
+	if _, ok := d.clusterEnv["RegistryDomain"]; !ok {
+		d.clusterEnv["RegistryDomain"] = "sea.hub"
```
Comment (VinceCui) [inline pkg/infradriver/ssh_infradriver.go:154]: Strings like "sea.hub", "5000", should be consts and put into well known common file.

File: pkg/infradriver/ssh_infradriver.go [hunk 3]
```diff
@@ -141,11 +142,21 @@ func (d *SSHInfraDriver) GetHostIPListByRole(role string) []net.IP {
 }
 
 func (d *SSHInfraDriver) GetHostEnv(host net.IP) map[string]interface{} {
-	return d.hostEnvMap[host.String()]
+	hostEnv := d.hostEnvMap[host.String()]
+	if _, ok := hostEnv["HostIP"]; !ok {
```
Comment (VinceCui) [inline pkg/infradriver/ssh_infradriver.go:146]: Keys like "HostIP" should be a const and put it into well known common file

**Review by VinceCui — COMMENTED** [review]
Attention UT.

**Review by kakaZhou719 — APPROVED** [review]
LGTM

**Pull request merged**